src/connectors/index.ts

```diff
@@ -10,6 +10,32 @@ export enum AdditionalChainIds {
     CELO = 42220,
 }
 
+const devEnv = ['development-celo', 'fuse', 'fuse-mainnet']
+const stagingEnv = ['staging-celo', 'staging', 'staging-mainnet']
+
+const getUrl = (env: string) => {
+    const walletEnv = devEnv.includes(env) ? 'dev' : stagingEnv.includes(env) ? 'qa' : 'wallet'
+    return `https://${walletEnv}.gooddollar.org/wc?uri=`
+}
+
+export const gd = customwc({
+    label: 'gooddollar',
+    ...(wc2InitOptions as any),
+    handleUri: (uri) =>
+        new Promise((res) => {
```

Comment:
you don't perform any async operations, why do you need promise ?
if it's just an handleUri interface requirement you could just declare 

```javascript
handleUri: async uri => {
  // do your stuff
  return true
}
```

Reply:
done